package.json

```diff
@@ -7,8 +7,8 @@
   "scripts": {
     "dev": "rm -rf lib && tsc --watch",
     "build": "rm -rf lib && tsc",
-    "test": "jest --config ./test/jest.json --runInBand",
-    "cover": "jest --config ./test/jest.json --runInBand --coverage",
```

Comment:
Did you just add `--bail` for testing CI failure? Can we remove please?

Reply:
Otherwise jest failes with exit code 0